constants/issues.ts

```diff
@@ -1,44 +1,3 @@
-export type IssueStatus = 'todo' | 'doing' | 'done'
-export interface Issue {
-  id: string
-  title: string
-  labels: IssueStatus[]
-}
+export const ISSUE_STATUSES = ['todo', 'doing', 'done'] as const
```

Comment:
I never understood that to be the case.

I understand that this might be the goal in the long term, but for this prototype doesn't it make more sense to have hardcoded columns as specified [here](https://github.com/orgs/cytechmobile/projects/11?pane=issue&itemId=44712607)?

This way there's no need to set up a repo with data labels beforehand.

Reply:
yes, hardcoded columns please.